crates/re_types/src/components/range1d_ext.rs

```diff
@@ -0,0 +1,20 @@
+use crate::datatypes;
+
+use super::Range1D;
+
+impl Range1D {
+    #[inline]
+    pub fn new(start: f64, end: f64) -> Self {
+        Self(datatypes::Range1D([start, end]))
+    }
+
+    #[inline]
+    pub fn start(&self) -> f64 {
+        self.0 .0[0]
+    }
+
+    #[inline]
+    pub fn end(&self) -> f64 {
+        self.0 .0[1]
+    }
```

Comment:
Do we really want to inherit the terrible names used by the rust ranges? 😬  I would personally prefer more symmetrical names like:

* begin/end
* start/stop
* min/max
* from/to

Reply:
🤷 I have no grudge against start/end. And I'd rather not derive from the pattern established in a standard library if I don't have to

Reply:
sadly I guess you are right